docs/conf.py

```diff
@@ -119,13 +119,22 @@
     'font': {
         'text': 'Open Sans'
     },
-    # BEGIN: version_dropdown
     'version_dropdown':
         True,
     'version_info': [
+        {
+            'version': 'https://kubeflow-pipelines.readthedocs.io/en/master/',
+            'title': 'master',
+            'aliases': [],
+        },
+        {
+            'version': 'https://kubeflow-pipelines.readthedocs.io/en/2.0.0b5/',
+            'title': 'v2.0.0b5',
+            'aliases': [],
+        },
         {
             'version': 'https://kubeflow-pipelines.readthedocs.io/en/2.0.0b4/',
-            'title': 'v2.0.0b4 [pre-release]',
```

Comment:
For my education what is the use of this label and why removing it?
Technically beta is still considered pre-release--pip install won't upgrade to a beta version unless add `--pre` flag.

Reply:
It's the label on the dropdown to select the version. This choice was to reduce verbosity from having several different pre versions all say `[pre-release]`, when this is implied by the `b` in the version.

Happy to re-add if you think being more explicit is better.